workflow.py

```
@@ -0,0 +1,325 @@
+
+    return result_text
+
+
+async def run_workflow(
+    requirement: str,
+    workflow: list[str],
+    is_bug_fix: bool = False,
+    start_from: str = "planner",
+):
+    """전체 워크플로우를 순서대로 실행한다."""
+    print(f"\n🚀 EEOS 개발 워크플로우 시작")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)